lua/luasnip/loaders/from_vscode.lua

```diff
@@ -131,7 +131,7 @@ local function load_snippet_folder(root, opts)
 	)
 end
 
-function filter_list(list, exclude, include)
+local function filter_list(list, exclude, include)
```

Comment:
This change is currently causing errors because the position of this function is too low in this file as it is used above this line (before it is defined).  This error is visible now, since (thankfully) it is no longer global @leiserfg or @L3MON4D3

Reply:
Sorry for the Verschlimmbesserung